plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -385,63 +381,92 @@ def _get_saved_streaming_attributes(self):
     def update_saved_streaming_attributes(self, attributes):
         Utils.write_json_to_file(self.streaming_attributes_file, attributes)
 
-    def _parse_telemetry_csv_metrics_to_json(self, data, line_separator="\n", attrs_separator=","):
+    def _get_filtered_counters(self, counters):
```

Comment:
let us document the function and what it does

Reply:
done